File: grupo_5_proyecto_2026_backend/FourLoveRestaurant/Models/Supplier.cs
using System;
using System.Collections.Generic;

namespace FourLoveRestaurant.Models;

public partial class Supplier
{
    public int Id { get; set; }

    public string? Photo { get; set; }

    public string? Name { get; set; }

    public string? Identification { get; set; }

    public bool? Isactive { get; set; }

    public string? Phone { get; set; }

    public string? Email { get; set; }

    public virtual ICollection<Ingredient> Ingredients { get; set; } = new List<Ingredient>();
}

File: grupo_5_proyecto_2026_fronted/src/types/Supplier.ts
export type Supplier = {
  id?: number;
  photo?: string;          // ruta  "/uploads/suppliers/..."
  name?: string;
  identification?: string;
  phone?: string;
  email?: string;
  isactive?: boolean;
};


export type SupplierFormData = {
  name: string;
  identification: string;
  phone: string;
  email: string;
  photoFile?: File | null;   
  photo?: string;            
};
